test/system/libs/helpers.bash

```diff
@@ -47,7 +47,7 @@ function _setup_environment() {
 }
 
 function _setup_containers_storage() {
-  mkdir -p ${TEMP_STORAGE_DIR}
+  mkdir -p ${TEMP_STORAGE_DIR} || true
```

Comment:
Is this really needed?

Reply:
It was just a try Matthias and I did, but did not work as expected.